shared/actions/platform-specific/index.desktop.tsx

```diff
@@ -404,12 +404,24 @@ const setOpenAtLogin = async (state: Container.TypedState) => {
     },
   })
 
-  if (!__DEV__ && SafeElectron.getApp().getLoginItemSettings().openAtLogin !== openAtLogin) {
```

Comment:
Sorry I missed the configGuiSetValueRpcPromise part above this. This doesn't work because the user could change their setting in the CLI which doesn't set this with `keybase ctl autostart`. On Linux can we derive the initial state from the file itself instead of this config value? And we don't need to set it either

Reply:
Is it ok to do RPC to service when the initialisation function runs? Is the service running and accepting RPC at that point? Confused because comments in e.g. `loadWindowState`.

Reply:
Can you do it when the settings tab/advanced tab is open? cc @buoyad

Reply:
For example see loadLockdownMode

Reply:
You're right @taruti it's probably not safe. @heronhaye's suggestion of loading when settings/advanced is opened sounds good.

Reply:
I have been thinking over this quite a lot.

Currently we support the cli non-advertised ctl only on Linux. Is there a reason why we want to support it on Linux in a different fashion from Windows and Mac? I can do the code which reduces the race between GUI and the cli command, but not sure if it is really worth the added Linux only complexity.

Reply:
I can't comment on how much more complex it is (cc @buoyad) but it seems reasonable that the implementation should be different if the inner workings are (xdg vs whatever the electron startup manager does).

Reply:
We can call the RPC on linux when settings/advanced is opened and use that to populate `state.config.openAtLogin`

Reply:
Ok, updated the PR to do that. Do you want first to look at the latest changes to the PR, or have it rebased (conflicts)?

Reply:
Now it calls the RPC on startup, is the connection ready when starting the app up normally?

Reply:
Oh I misread that, nevermind. LGTM